spec/jobs/education_form/create_spool_submissions_report_spec.rb

```diff
@@ -64,6 +64,26 @@ def perform
           expect(File.read(filename)).to eq(csv_string)
         end
 
+        it 'should check for stem submissions' do
```

Comment:
These are testing the results of `#create_csv_array`, so you can move them under the 
> describe '#create_csv_array' do

section and you can remove the `perform` method calls.  They're not necessary.

I also noticed an indentation issue that you didn't create.
`describe '#perform'` should be at the same level of indentation as `describe '#create_csv_array'`.
Currently it's nested inside it.
Can you fix that too please while you're in here?

Reply:
Thanks, done!